Features/GoogleAnalytics/script.twig

```diff
@@ -0,0 +1,17 @@
+<script>
+  {% if trackingEnabled %}
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+  {% else %}
+    function ga() {
+      console.log('GoogleAnalytics: ' + [].slice.call(arguments));
+    }
+  {% endif %}
+  var gaId = {{ gaId|json_encode|raw }}
```

Comment:
Doesn't have to be wrapped, the filters take care of that. I added the missing semi-colon.